### ISSUE TITLE
detection not proper way

### ISSUE THREAD

**Issue by narasimharaovadlamudi** [cvetaevvitaliy/rockchip_yolov5_npu#2]
steps i followed::

!git clone https://github.com/cvetaevvitaliy/rockchip_yolov5_npu  # clone

%cd rockchip_yolov5_npu/yolov5

%pip install -qr requirements.txt  # install

import torch
import utils
display = utils.notebook_init()

!curl -L -o yolov5s.pt https://github.com/ultralytics/yolov5/releases/download/v6.0/yolov5s.pt

!python export.py --weights yolov5s.pt --imgsz 640 640 --batch 1 --device cpu --include onnx --rknpu RK3588

 python convert.py ../model/present.onnx rk3588
I rknn-toolkit2 version: 2.3.2
--> Config model
done
--> Loading model
I Loading : 100%|██████████████████████████████████████████████| 120/120 [00:00<00:00, 12096.34it/s]
done
--> Building model
I OpFusing 0: 100%|█████████████████████████████████████████████| 100/100 [00:00<00:00, 1662.35it/s]
I OpFusing 1 : 100%|████████████████████████████████████████████| 100/100 [00:00<00:00, 1238.49it/s]
I OpFusing 2 : 100%|█████████████████████████████████████████████| 100/100 [00:00<00:00, 132.49it/s]
W build: found outlier value, this may affect quantization accuracy
                        const name               abs_mean    abs_std     outlier value
                        model.0.conv.weight      1.47        2.85        28.488
I GraphPreparing : 100%|███████████████████████████████████████| 142/142 [00:00<00:00, 10918.46it/s]
I Quantizating : 100%|████████████████████████████████████████████| 142/142 [00:10<00:00, 14.11it/s]
W build: The default input dtype of 'images' is changed from 'float32' to 'int8' in rknn model for performance!
                       Please take care of this change when deploy rknn model with Runtime API!
W build: The default output dtype of 'output' is changed from 'float32' to 'int8' in rknn model for performance!
                      Please take care of this change when deploy rknn model with Runtime API!
W build: The default output dtype of '354' is changed from 'float32' to 'int8' in rknn model for performance!
                      Please take care of this change when deploy rknn model with Runtime API!
W build: The default output dtype of '355' is changed from 'float32' to 'int8' in rknn model for performance!
                      Please take care of this change when deploy rknn model with Runtime API!
I rknn building ...
I rknn building done.
done
--> Export rknn model
done

<img width="640" height="640" alt="Image" src="https://github.com/user-attachments/assets/04ae09e3-06c3-4acf-a851-e200e47c5523" />

please share how to reslove?

Thank you.
